### PR TITLE
Create PRS in BLorient5268 (owners ms and ms-part/p4)

### PR DESCRIPTION
**Pull request by CarstenHoffmannMarburg**

### CODE REVIEW

File: new/PRS14371Swaynn.xml [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+        <revisionDesc>
+            <change who="CH" when="2024-03-15">Created entity</change>
+        </revisionDesc>
+    </teiHeader>
+    <text>
+        <body>
+            <listPerson>
+                <person sex="1">
+                    <persName xml:lang="en" xml:id="n1"><forename>H.</forename> <forename>G.</forename> 
+                        <forename>C.</forename> <surname>Swaynn</surname></persName>
```
Comment (eu-genia) [inline new/PRS14371Swaynn.xml:48]: Swayne

Comment (eu-genia) [reply new/PRS14371Swaynn.xml:48]: https://www.abhilekh-patal.in/jspui/handle/123456789/4159727?searchWord=&backquery=[query=&filter_field_1=subjectCollection&filter_type_1=equals&filter_value_1=Digitized+Public+Records&filter_field_2=keywords&filter_type_2=equals&filter_value_2=Bombay&originalquery=&sort_by=dc.date.accessioned_dt&order=desc&rpp=20&etal=0&start=40]

https://en.wikipedia.org/wiki/Harald_George_Carlos_Swayne

File: new/PRS14371Swaynn.xml [hunk 2]
```diff
@@ -0,0 +1,60 @@
+<?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+schematypens="http://relaxng.org/ns/structure/1.0"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+<TEI xmlns="http://www.tei-c.org/ns/1.0" xml:id="PRS14371Swaynn" xml:lang="en" type="pers">
+    <teiHeader>
+        <fileDesc>
+            <titleStmt>
+                <title>H. G. C. Swaynn</title>
```
Comment (eu-genia) [inline new/PRS14371Swaynn.xml:8]: Swayne

File: new/PRS14370Dabasey.xml [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+        <revisionDesc>
+            <change who="CH" when="2024-03-15">Created entity</change>
+        </revisionDesc>
+    </teiHeader>
+    <text>
+        <body>
+            <listPerson>
+                <person>
+                    <persName xml:lang="gez" xml:id="n1">ደበስይ፡</persName>
+                    <persName xml:lang="gez" type="normalized" corresp="#n1">Dabasǝy</persName>
```
Comment (eu-genia) [inline new/PRS14370Dabasey.xml:49]: no date?

**Review by eu-genia — CHANGES_REQUESTED** [review]

File: new/PRS14371Swaynn.xml (hunk 1 above)
Comment (CarstenHoffmannMarburg) [reply new/PRS14371Swaynn.xml:48]: It is Capt. Swaynn in the catalogue description. It is maybe a misreading by Strelcyn. I think, it is reasonable to change to Swayne. I will notify the difference in the manuscript record in a note.

grafik

**Review by eu-genia — APPROVED** [review]

**Pull request merged**